documentation/modules/ROOT/pages/operations/quarkus-debezium-engine-extension.adoc

```diff
@@ -0,0 +1,422 @@
+quarkus.datasource.username=<your username>
+quarkus.datasource.password=<your password>
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/hibernate_orm_test
+quarkus.datasource.jdbc.max-size=16
+```
+
+The configuration parameters available are in the https://debezium.io/documentation/reference/stable[Debezium documentation]. Additionally, you must specify the https://quarkus.io/guides/datasource#configure-a-jdbc-datasource[datasource configuration parameters] as required by {prodname} runtime.
+
+
+== Capturing events from {prodname}
+
+Continuing from the previous minimal configuration, your Quarkus application can receive CDC event payload directly:
+
+```java
+import io.debezium.runtime.CapturingEvent;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.apache.kafka.connect.source.SourceRecord;
+
+import io.debezium.runtime.Capturing;
+
+@ApplicationScoped
+public class ProductHandler {
+
+
+    @Capturing
+    public void capture(CapturingEvent<SourceRecord> record) {
+        // process your events
+    }
+
+}
+```
+
+The `CapturingEvent<T>` contains information related to the kind of database operation:
+
+```java
+    @Capturing
+    public void capture(CapturingEvent<SourceRecord> record) {
+        switch (record) {
+            case Create<SourceRecord> event -> {}
+            case Delete<SourceRecord> event -> {}
+            case Message<SourceRecord> event -> {}
+            case Read<SourceRecord> event -> {}
+            case Truncate<SourceRecord> event -> {}
+            case Update<SourceRecord> event -> {}
+        }
+    }
+```
+
+=== Filtering Change Data Capture Events
+
+It's possible to filter events by `destination`:
+
+```java
+    @Capturing(destination = "native.inventory.products")
+    public void capture(CapturingEvent<SourceRecord> record) {
+        // process your event
+    }
+```
+
+The default behavior is that a Debezium connector `destination` is formed from the name of the `prefix` defined in the configuration with the database name and the name of the table in which the change was made. In some cases the `destination` is redefined using an https://debezium.io/documentation/reference/stable/transformations/topic-routing.html[SMT].
+
+=== Deserializer via Jackson
+
+Quarkus has built-in support for JSON serialization and deserialization based on Jackson. There is an existing `ObjectMapperDeserializer` that can be used to deserialize all data objects via Jackson.
+
+The corresponding deserializer class needs to be subclassed. So, we have to create a `ProductDeserializer` that extends the `ObjectMapperDeserializer`.
+
+```java
+public class ProductDeserializer extends ObjectMapperDeserializer<Product> {
+    public ProductDeserializer() {
+        super(Product.class);
+    }
+}
+```
+
+Finally, configure your capture channel to use the Jackson deserializer for a particular destination:
+
+```properties
+quarkus.debezium.capturing.products.destination=native.inventory.products
```

Comment:
Maybe I will clarify that the matching with the `handler` is made with the `destination` value.